CHANGELOG.md

```diff
@@ -18,6 +18,17 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Changes the YouTube video link for authentication tutorial (the old video was in low definition, the new one is in high definition)
 - Updated links to Spotify in documentation 
 
+## [2.23.1] - 2023-12-28
```

Comment:
2.23.1 isn't released yet, can you merge this section with `Unreleased` above?

Reply:
Thanks for your advice. I moved it to `Unreleased` as you advised already.